tests/bears/BearTest.py

```diff
@@ -129,6 +140,24 @@ def test_bad_bear(self):
         # debug message contains custom content, dont test this here
         self.queue.get()
 
+    def test_LocalBear(self):  # Fails because of no run() implementation
```

Comment:
I renamed them as `test_print_filename_LocalBear` and `test_print_filename_GlobalBear` but we don't print filenames for the GlobalBear ? so should I rename it to something else or are the changes good ? @Makman2